pysatNASA/instruments/maven_insitu_kp.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Module for the MAVEN insitu instruments.
+
+Supports the in situ Key Parameter (kp) data from multiple instruments
+onboard the Mars Atmosphere and Volatile Evolution (MAVEN) satellite.
+
+Accesses local data in CDF format.
+Downloads from CDAWeb.
+
+Properties
+----------
+platform
+    'maven'
+name
+    'insitu_kp'
+tag
+    None supported
+inst_id
+    None supported
+
+
+Examples
+--------
+::
+    import pysat
+
+    insitu = pysat.Instrument(platform='maven', name='insitu_kp')
```

Comment:
```suggestion
    insitukp = pysat.Instrument(platform='maven', name='insitu_kp')
```

or

 ```suggestion
    kp = pysat.Instrument(platform='maven', name='insitu_kp')
```
or
```suggestion
    insitukp = pysat.Instrument(platform='maven', name='insitu_kp')
```


for general consistency with a later comment on a different instrument that also used insitu.